track_analyzing/track_analyzer/cmd_table.cpp

```diff
@@ -450,16 +450,12 @@ void CmdTagsTable(string const & filepath, string const & trackExtension, String
           }
 
           // If it's not the end of the track than it could be a crossroad.
-          IsCrossroadChecker::CrossroadInfo crossroad{};
           if (end != track.end())
-          {
-            crossroad = checker(prev->GetSegment(), end->GetSegment());
-            IsCrossroadChecker::MergeCrossroads(crossroad, info);
-          }
+            IsCrossroadChecker::MergeCrossroads(checker(prev->GetSegment(), end->GetSegment()), info);
```

Comment:
clang format

Reply:
Too long line? Indent is ok, it's after if.

But done anyway.
```c++
          if (end != track.end())
          {
            IsCrossroadChecker::MergeCrossroads(checker(prev->GetSegment(), end->GetSegment()),
                                                info);
          }
```